meta-client/src/test/java/org/apache/helix/metaclient/impl/zk/TestZkMetaClient.java

```diff
@@ -0,0 +1,70 @@
+package org.apache.helix.metaclient.impl.zk;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import java.io.File;
+import java.io.IOException;
+import org.apache.commons.io.FileUtils;
+import org.apache.helix.zookeeper.zkclient.IDefaultNameSpace;
+import org.apache.helix.zookeeper.zkclient.ZkServer;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+
+public class TestZkMetaClient {
+
+  private static final String ZK_ADDR = "localhost:2183";
```

Comment:
Neat: May be you already have this mind but just for my understanding this is something as ZKTestBase right where all ZK-MetaClient related test cases will be using base methods and configs from this this class? 
Also curious about zookeeper port, usually default port for zookeeper server is "2181" by default, is this the reason we selected different port to avoid using same port?

Reply:
Thanks for the comments. You are right on both points. In the future, yes, it's likely there will be a base test class or singleton for zkclient, but that will need some more thoughts and refactoring.  
Also I chose a different port number to differentiate.